.github/workflows/ocp-self-runner.yml

```diff
@@ -0,0 +1,111 @@
+name: Test self hosted runner
+
+on:
+  workflow_call:
+  # So we can trigger manually if needed
+  workflow_dispatch:
+
+jobs:
+  ginkgo-test:
+    name: ginkgo tests
+    runs-on: self-hosted
+    steps:
+      - uses: actions/checkout@v4.1.1
+      
+      - uses: actions/setup-go@v5
+        with:
+          go-version: v1.20
+          cache: true
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v4
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Install kubectl
+        uses: azure/setup-kubectl@v4
+        id: install
+
+      - name: Install dependencies
+        run: |
+          export USE_SUDO=false
+          mkdir clusteradm
+          export INSTALL_DIR="$PWD/clusteradm"
+          curl -L https://raw.githubusercontent.com/open-cluster-management-io/clusteradm/main/install.sh | bash
+          curl -L https://github.com/kubestellar/kubeflex/releases/download/v0.4.2/kubeflex_0.4.2_linux_amd64.tar.gz --output kubeflex.tar.gz
+          tar -xvf kubeflex.tar.gz bin/kflex
+          go install github.com/onsi/ginkgo/v2/ginkgo
+          curl -L https://mirror.openshift.com/pub/openshift-v4/clients/ocp/stable-4.15/openshift-client-linux.tar.gz --output openshift-client-linux.tar.gz
+          tar -xvf openshift-client-linux.tar.gz 
+          cp oc bin
+          curl -L https://github.com/kubestellar/kubestellar/archive/refs/tags/v0.22.0.tar.gz --output kubestellar.tar.gz
+          tar -xvf kubestellar.tar.gz 
+          mv kubestellar-0.22.0 kubestellar
+
+      - name: Run test
+        env:
+          TEST_FLAGS: ${{ github.event.inputs.testFlags }}
```

Comment:
This workflow does not define any inputs.

Reply:
I removed the TEST_FLAGS and the env section.